### PR TITLE
Adding support for Windows high-contrast

### PR DESCRIPTION
**Pull request by sdule**
# Guide

**This "Help" section can be deleted before submitting this pull request.**

*Update the name of this pull request to reflect the following shape:*

```
{type}/{scope?}/{message}
```

* **type** - A [conventional commit type](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional#type-enum) **REQUIRED**
* **scope** - The kabab-case scope of the changes in this request
* **message** - A short, kebab-case statement describing the changes **REQUIRED**

*Provide a general summary of the scope of the changes in this pull request.*

* [Readme](https://github.com/momentum-design/tokens/blob/master/README.md)
* [Getting Started Guide](https://github.com/momentum-design/tokens/blob/master/GETTING_STARTED.md)
* [Contributing Guide](https://github.com/momentum-design/tokens/blob/master/CONTRIBUTING.md)

# Description

Adding the support for Windows high-contrast theme within the momentum system. A new theme was created, that instead to color tokens, maps to Microsoft-defined high-contrast tokens, that can be then resolved at runtime, by reading Windows registry values from the app itself.

A new token was defined for the button border, since we now need that level of indirection, as regular themes want it at 'none' while high-contrast needs an actual value there. So the border definintion was moved from components to theme files.

Some changes were made to the theme generator script as follows:
- cmd line switch `winSystemHighContrast` that needs to be used if we want to work with Win HC themes
- skipping parsing of core tokens if the above is set
- for 'terminal' tokens that don't need to be resolved, we now check if they conform to the HC specification, and if they do, we remove the `SystemColor` part when generating a theme, as it's pretty redundant.

NB - as we don't currently have all the specs, I've left a `PlaceholderColor` token in, which in our implementation resolves to the default background color, but others are of course free to do a different mapping.

# Links

Definition of high-contrast colors and themes from Microsoft: https://docs.microsoft.com/en-us/windows/apps/design/accessibility/high-contrast-themes#high-contrast-colors

### CODE REVIEW

File: platformcomponents/desktop/checkbox.json [hunk 1]
```diff
@@ -5,25 +5,25 @@
       "#normal":{
         "background": "@theme-control-active-normal",
         "text": "@theme-text-primary-normal",
-        "border": "none",
+        "border": "@theme-button-border",
```
Comment (sdule) [inline platformcomponents/desktop/checkbox.json:8]: Do we want component-specific `border` tokens here?

**Comment by jimmycoyne** [comment]
@sdule should we  generate the HC output any time we select the windows platform ?

**Comment by sdule** [comment]
@jimmycoyne so I'm not sure if platform today influences which themes would be generated, rather than just details during generation? If it does, I don't mind adding it

**Comment by Coread** [comment]
CI and package.json changes LGTM

**Review by Coread — APPROVED** [review]

**Review by paulwitty — APPROVED** [review]

**Pull request merged**